tests/test_ag3.py

```diff
@@ -358,6 +358,44 @@ def test_snp_sites(chunks, region):
         assert d.shape == d.compute().shape
 
 
+@pytest.mark.parametrize("chrom", ["2RL", "3RL"])
+def test_snp_sites_for_contig(chrom):
+    ag3 = setup_ag3()
+    contig_r = chrom[0] + chrom[1]
+    contig_l = chrom[0] + chrom[2]
+    sites_r = ag3.snp_sites(region=contig_r, field="POS")
+    sites_l = ag3.snp_sites(region=contig_l, field="POS")
+    max_r = ag3.genome_sequence(region=contig_r).shape[0]
+    sites = da.concatenate([sites_r, sites_l + max_r])
+
+    sites_joined = ag3.snp_sites(region=chrom, field="POS")
```

Comment:
Very small nit, I'd be inclined to rename `sites` to `sites_expected`, and rename `sites_joined` to `sites_actual`. I find it often useful to use this "expected"/"actual" naming convention to help keep track of what's what in tests.